sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AQEOptimizer.scala

```diff
@@ -27,9 +28,15 @@ import org.apache.spark.util.Utils
  * The optimizer for re-optimizing the logical plan used by AdaptiveSparkPlanExec.
  */
 class AQEOptimizer(conf: SQLConf) extends RuleExecutor[LogicalPlan] {
+  private def fixedPoint =
+    FixedPoint(
+      conf.optimizerMaxIterations,
```

Comment:
do we need a separate conf for `AQEOptimizer`

Reply:
I don't see a user request at least until now